File: experiment/java-python/valid/j2pyrawpy/for__1__MAXIMUM_PRODUCT_INCREASING_SUBSEQUENCE.py
#!/usr/bin/env python
""" generated source for module for__1__MAXIMUM_PRODUCT_INCREASING_SUBSEQUENCE """
class X(object):
    """ generated source for class X """
    @classmethod
    def lis(cls, arr, n):
        """ generated source for method lis """
        mpis = [None]*n
        max = Integer.MIN_VALUE
        k = 0
        while k < mpis.length:
            if mpis[k] > max:
                max = mpis[k]
            k += 1
        return max

File: experiment_jp_valid/mutants/for__1__MAXIMUM_PRODUCT_INCREASING_SUBSEQUENCE.java
static int lis ( int [ ] arr , int n ) { int [ ] mpis = new int [ n ] ; int max = Integer . MIN_VALUE ; for ( int k = 0 ; k < mpis . length ; k ++ ) { if ( mpis [ k ] > max ) { max = mpis [ k ] ; } } return max ; }
